cluster/v1alpha1/types.go

```diff
@@ -233,6 +233,15 @@ type PlacementSpec struct {
 	// Predicates represent a slice of predicates to select ManagedClusters. The predicates are ORed.
 	// +optional
 	Predicates []ClusterPredicate `json:"predicates,omitempty"`
+
+	// PrioritizerConfigs defines the configurations of the prioritizers.
+	// If this field is unset, i.e. nil, then default configuration is used, in which
+	// Steady and Balance prioritizers have the weight of 1 while other prioritizers have the weight of 0.
+	// If you don't want to use any prioritizer, set this field to an empty slice, i.e. `prioritizerConfigs: []`
+	// in the YAML file, then all the prioritizers will be disabled.
```

Comment:
You should explicitly indicate that once a single prioritizer has been configured, then every desired prioritizer *must* be enumerated.

I suggest you make a note in your enhancement that this behavior should be reassessed for whether it has worked well before beta.

It is also possible that you'll want a config like

```
PrioritizerConfig PrioritizerConfig

PrioritizerConfig struct{
  // mode is either Exact, Additive, "" where "" is Additive by default.
  // in Additive mode, any prioritizer not explicitly enumerated is enabled in its default mode
  // in Exact mode, any prioritizer not explicitly enumerated is weighted as zero
  // Exact requires knowing the full set of prioritizers you want, but avoids behavior changes between releases
  // Additive doesn't require configuring all prioritizers.  As new prioritizers are added, additional
  // prioritization will happen
  Mode string

  PrioritizerConfigs []PrioritizerConfig
}

```

Reply:
@deads2k thanks, this makes sense to us.